docs/static/java-codec.asciidoc

```diff
@@ -376,11 +318,11 @@ To test the plugin, start Logstash with:
 
 [source,java]
 -----
-echo "foo,bar" | bin/logstash --java-execution -e 'input { java_stdin { codec => java_codec_example } } }'
+echo "foo,bar" | bin/logstash -e 'input { java_stdin { codec => java_codec_example } } }'
 -----
 
-NOTE: The `--java-execution` flag to enable the Java execution engine is required as Java plugins are not supported
-in the Ruby execution engine.
+NOTE: The Java execution engine, the default execution engine since Logstash 7.0, is required
```

Comment:
Previously, attempting to run a Java plugin in the Ruby execution engine would fail somewhere in the name resolution or plugin instantiation phase with a potentially misleading error message. I've added an explicit check for that with a much clearer error message:

`Cannot start the Java plugin 'foo' in the Ruby execution engine. The Java execution engine is required to run Java plugins.`